### PR TITLE
Add styles for the switch and checkbox widget

### PR DESCRIPTION
**Pull request by vinothvino42**
Ticket: #964

In this PR, I just removed the `form_checkbox.dart` and created two widgets called `Switch` and `Checkbox`

**Sample EDL:**

```yaml
View:
  header:
    title: Switch & Checkbox
  body:
    Column:
      styles: { gap: 24, padding: 24, scrollable: true }
      children:
        - Checkbox:
            leadingText: Leading text
            styles:
              {
                selectedColor: yellow,
                unselectedColor: blue,
                checkColor: pink,
              }

        - Switch:
            leadingText: Set to true and requred
            required: true
            value: true
            styles:
              {
                activeColor: yellow,
                activeThumbColor: black,
                inactiveColor: blue,
                inactiveThumbColor: grey,
              }

```

**Selected State:**

**Unselected State:**

### CODE REVIEW

File: lib/widget/input/form_checkbox.dart [hunk 1]
```diff
@@ -149,19 +162,53 @@ class OnOffState extends FormFieldWidgetState<OnOffWidget> {
   /// we adjust the hit area of the checkbox here. 40px is smaller than the default (48px)
   /// but it seem to be reasonable for touch device, plus the alignment inside
   /// form is much better (align well with the rest of the input widgets)
+
   Widget get aCheckbox {
     return SizedBox(
         width: 40,
         height: 40,
         child: Checkbox(
+            side: widget._controller.inactiveColor != null
```
Comment (vusters) [inline lib/widget/input/form_checkbox.dart:171]: Why are we decorating the border if it's inActive? The user sets inActive thinking it's equivalent to !active, but instead the fillColor stays the same but they get a different border? This doesn't make sense.

File: assets/schema/ensemble_schema.json [hunk 2]
```diff
@@ -1569,6 +1569,26 @@
             },
             "trailingText": {
               "type": "string"
+            },
+            "activeColor": {
```
Comment (vusters) [inline assets/schema/ensemble_schema.json:1573]: Please don't include all properties that don't make sense. Break these up. Each widget should only get properties that apply to them.

**Review by vusters — CHANGES_REQUESTED** [review]

File: assets/schema/ensemble_schema.json (hunk 2 above)
Comment (vinothvino42) [reply assets/schema/ensemble_schema.json]: Can you read the above description of the PR? I just updated

File: lib/widget/input/form_checkbox.dart (hunk 1 above)
Comment (vinothvino42) [reply lib/widget/input/form_checkbox.dart]: Can you read the above description of the PR? I just updated

**Comment by vinothvino42** [comment]
@vusters I split this into two separate files for two widgets (switch and checkbox) with their own controller and styles. Can you review it?

**Review by vusters — APPROVED** [review]

**Pull request merged**